pkgs/development/python-modules/datetime/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytz
+, zope_interface
+}:
+
+buildPythonPackage rec {
+  pname = "datetime";
+  version = "5.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "zopefoundation";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-IxJwlJnSOo4CvEENm/0OJPwV4tLK/rU1fgFAsJEVDNU=";
+  };
+
+  propagatedBuildInputs = [
+    pytz
+    zope_interface
+  ];
+
+  pythonImportsCheck = [
+    "DateTime"
+  ];
+
+  meta = with lib; {
+    description = "DateTime data type, as known from Zope";
+    homepage = "https://github.com/zopefoundation/DateTime";
```

```suggestion
    homepage = "https://github.com/zopefoundation/DateTime";
    changelog = "https://github.com/zopefoundation/DateTime/releases/tag/${version}";
```
It is preferred to set the changelog.